src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3866,72 +3864,38 @@ namespace SimAirServingZones {
 		// Uses data from System Sizing input and the system to zone connection data
 		// calculated in InitAirLoops and stored in AirToZoneNodeInfo in DataLoopNode..
 
-		// Using/Aliasing
-		using General::FindNumberInList;
-		using Psychrometrics::PsyRhoAirFnPbTdbW;
-		using namespace OutputReportPredefined;
-		using DataHeatBalance::Zone;
-		using namespace DataSizing;
 
-		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-		int AirLoopNum; // primary air system index
-		int SysSizNum; // system sizing input index
-		int DesDayEnvrnNum; // design day index
-		int NumZonesCooled; // number of zones cooled by a system
-		int NumZonesHeated; // number of zones heated by a system
-		Real64 MinOAFlow; // design minimum outside air flow for a system
-		int ZonesCooledNum; // loop index over zones cooled by a system
-		int ZonesHeatedNum; // loop index over zones heated by a system
-		int CtrlZoneNum; // loop index over all controlled zones
-		int MatchingCooledZoneNum; // the cooled zone index matching a heated zone in a system
-		int PrimAirIndex; // index of System Sizing primary air system name in primary air system array
-		int SysSizIndex; // system sizing do loop index
 		static bool ErrorsFound( false ); // Set to true if errors in input, fatal at end of routine
-		Real64 ZoneOAFracCooling; // zone OA fraction for cooling design air flow
-		Real64 ZoneOAFracHeating; // zone OA fraction for heating design air flow
-		static Real64 Ep( 1.0 ); // zone primary air fraction
-		Real64 ZoneSA; // Zone supply air flow rate
-		Real64 ZonePA; // Zone primary air flow rate
-		Real64 ClgSupplyAirAdjustFactor; // temporary variable
-		Real64 HtgSupplyAirAdjustFactor; // temporary variable
-		Real64 SysOAUnc; // uncorrected system OA summing up people and area based OA for all zones for VRP
-		Real64 ZoneOAUnc; // uncorrected zone OA summing up people and area based OA for each zone
 
 		//have moved a large number of std 62.1 variables to DataSizing.hh so they can be used outside of this routine
 
 		// allocate arrays used to store values for standard 62.1 tabular report
 		if ( ! allocated( VpzClgByZone ) ) {
-			DataSizing::VdzClgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VdzMinClgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VdzHtgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VdzMinHtgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::ZdzClgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::ZdzHtgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VpzClgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VpzMinClgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VpzHtgByZone.dimension( NumOfZones, 0.0 );
-			DataSizing::VpzMinHtgByZone.dimension( NumOfZones, 0.0 );
+			DataSizing::VdzClgByZone.dimension( NumAirTerminalUnits, 0.0 );
```

Comment:
Unfortunate that all these array variables are named *ByZone when now they should be *ByTerminal or something.

Reply:
True, I could rename them if you think it's more clear.